cmd/osmosisd/cmd/init.go

```diff
@@ -80,13 +80,17 @@ func InitCmd(mbm module.BasicManager, defaultNodeHome string) *cobra.Command {
 			serverCtx := server.GetServerContextFromCmd(cmd)
 			config := serverCtx.Config
 
+			//Override settings in config.toml
+			config.P2P.Seeds = "085f62d67bbf9c501e8ac84d4533440a1eef6c45@95.217.196.54:26656"
```

Comment:
After talking to folks on Interchain Gmbh & Informal teams, I'm a bit worried about adding a default list of seeds for the following two reasons:

1) If you connect to a seed node, that itself doesn't consider itself to be a seed node, then you will remain connected to that node as a peer (not disconnect). So suppose we have N seeds listed, and `k` were malicious. As long as one of the seeds isn't malicious, I will get peers honestly on the network. However, in the current Tendermint, if those seeds are maliciously misconfigured to not disconnect from you, then they will fill up all your node connections. Then you won't sample any of the other peers you got from the honest seeds, and will just be connected to those `k` malicious seeds, thereby ruining your security.

2) Having a default seed list is great, as long as Tendermint itself is making random sampling guarantees. E.g. if I list 100 seeds, it should do one of:
 -  Get peers from each seed, build a large list, then randomly sample from the large list. 
 - Randomly sample which seeds to query, then build a sufficiently sized peer list. Then randomly sample peers of the local address book to go ping.

Reply:
I view (1) as a bug that needs to be fixed before I feel comfortable with this. 

I view (2) as something that I need to look into to double check, and if whats implemented is one of those scenarios, then lets go with this!

Reply:
Granted, there is an argument to be made that anything done with many validators putting things into this repo is much better than the currently mutable-by-anyone google sheets & hackmd's get passed around.

Thoughts @sunnya97 @UnityChaos ?

Reply:
Actually, I think (1) is fine by ensuring that we only seed nodes who we know the team for. This is similar to what bitcoin does: https://en.bitcoin.it/wiki/Satoshi_Client_Node_Discovery (Only 6 core devs curate lists) Thanks @sunnya97 for bringing this up.

We should still get that bug fixed tho.

(2) is something we should ensure, but less concerning.

Reply:
Yeah, so that is exactly what I am thinking:

we should know the operators.

Reply:
Yeah, I think well known validator operators making PRs to add their seed nodes would be fine. :+1: